### PR TITLE
Added index.html to fan-page

### PR DESCRIPTION
**Pull request by MVWalters**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	 <h2>Things to Consider?</h2>
+	  <dl>
+	 <dt>Where Should you Hike?</dt>
+	 <dd>While you can hike just about anywhere I'd suggest sticking to public lands to avoid conflicts with private land owners. State forests and state parks are a wonderful resource open to the public. Most have hiking trails open year round though you may want to check in the winter as some trails are converted for cross counrty skiing. Info on you local state parks and forests can be found here <a href="http://www.stateparks.com">www.stateparks.com</a>.</dd>
+	 <dt>What should you bring along?</dt>
+	 <dd>This depends on several factors including where you are hiking, the weather, and how long you plan to be on the trail. Somethings will always be a good idea to bring like water, a map, a camera, and a cell phone just incase something happens(better to be safe than sorry).</dd></dl>
+	 <p><strong>REMEMBER</strong>, it is always a good idea to tell someone where you are going and when to expect you back. The best places to hike are also the most remote and infrequently visited.</p>
+	</main>
+	</body>
+	<footer>
```
Comment (mbMosman) [inline index.html]: The footer needs to be inside the body tag.

**Comment by MVWalters** [comment]
How do you do a new pull request? The pull request from the previous weeks fan-page submission was still open when I went to do a screen shot this week

**Comment by mbMosman** [comment]
If it is still open, then just screenshot that one.  I might have missed closing it.


Mary Mosman

Computer Careers Faculty

Brooklyn Park Campus, 9000 Brooklyn Boulevard, Brooklyn Park, MN 55445

Eden Prairie Campus, 13100 College View Drive, Eden Prairie, MN 55347

HennepinTech.edu<https://mail.hennepintech.edu/owa/redir.aspx?SURL=X9_UXdraVORHgBEDAbJjlW7lFDPLHRjN-BS9r4ndo2C1J6JJO7nSCGgAdAB0AHAAcwA6AC8ALwBoAGUAbgBuAGUAcABpAG4AdABlAGMAaAAuAGUAZAB1AC8A&URL=https%3a%2f%2fhennepintech.edu%2f>

Support the HTC Foundation by using AmazonSmile<https://mail.hennepintech.edu/owa/redir.aspx?SURL=mTQW3FiySLMvYJeLHK4cUN5xDY1QSOqoOR3YmKzO9qu1J6JJO7nSCGgAdAB0AHAAOgAvAC8AcwBtAGkAbABlAC4AYQBtAGEAegBvAG4ALgBjAG8AbQAvAGMAaAAvADQAMQAtADEAOQA0ADMAMQA0ADkA&URL=http%3a%2f%2fsmile.amazon.com%2fch%2f41-1943149>

________________________________

**Comment by MVWalters** [comment]
Ok that must have been the case cause I turned it in early and now it looks like I can do a new pull request

**Pull request closed without merging**